gui.lua

```diff
@@ -272,9 +273,10 @@ function mail.show_message(name, msgnumber)
 	local from = minetest.formspec_escape(message.sender) or ""
 	local to = minetest.formspec_escape(message.to) or ""
 	local cc = minetest.formspec_escape(message.cc) or ""
+	local date = minetest.formspec_escape(os.date("%Y-%m-%d %X", message.time)) or ""
```

Comment:
There's been some issues caused by invalid or missing fields.
Not sure if this might be still issue or if it is fixed completely, issue might possible be there if "database" is very old?

To make it safe could make sure that  message.time is number (os.date will crash if not except for nil which returns current time):
```suggestion
	local date = type(message.time) == "number"
		and minetest.formspec_escape(os.date("%Y-%m-%d %X", message.time)) or ""
```